src/Controllers/Ua/StudyDocumentController.php

```diff
@@ -97,23 +105,27 @@ public function createDocument(Request $request, Response $response, array $args
      * @param Response $response
      * @param array $args
      * @return Response
-     * @throws \Keros\Error\KerosException
+     * @throws KerosException
      */
     public function getDocument(Request $request, Response $response, array $args)
     {
         $this->logger->debug("Getting document path for study " . $args["studyId"] . " and document type " . $args['documentId'] . " from " . $request->getServerParams()["REMOTE_ADDR"]);
 
         $document = $this->studyDocumentService->getLatestDocumentFromStudyDocumentType($args["studyId"], $args['documentId']);
 
-        return $response->withJson(array('location' => $this->kerosConfig['BACK_URL'] . DIRECTORY_SEPARATOR . $this->kerosConfig['STUDY_DOCUMENT_DIRECTORY'] . $document->getLocation()), 200);
+        $location = $this->directoryManager->uniqueFilename($document->getLocation(), false, $this->kerosConfig['TEMPORARY_DIRECTORY']);
+
+        $this->directoryManager->symlink('../../' . $this->kerosConfig['STUDY_DOCUMENT_DIRECTORY'] . $document->getLocation(), $location);
```

Comment:
Enlever les ../../ je pense. `$this->kerosConfig['STUDY_DOCUMENT_DIRECTORY']` sera un chemin absolu en DEV et PROD, mais peut contenir les ../../ enlocal pour tester si tu veux. 
Par principe, évitez les `..` dans le code, niveau sécu

Reply:
C'est possible en local de mettre `../../` devant en les path dans le setting, les seuls problèmes c'est que pour les tests ça va créer les dossiers hors du projet Keros-Back et le lien symbolique n'est pas valide. Du coup je sais pas si ça fonctionnera à 100% sur le serv de DEV
Après c'est sûr le `../..` dans le code est vrai pas ouf. Faudrait, je sais pas, pouvoir récup la partie commune des path en absolu